vsphere/vsphere.go

```diff
@@ -83,10 +83,19 @@ func NewClient(cfg *VSphereConfig) (cloudops.Ops, error) {
 	logrus.Debugf("  Datacenter: %s", vmObj.Datacenter.Name())
 	logrus.Debugf("  VMUUID: %s", cfg.VMUUID)
 
-	configMap, err := configmap.New(vSphereDataStoreLock, nil, 0, 420, 5*time.Second, 10*time.Second)
-	if err != nil {
-		logrus.Errorf(err.Error())
-		return nil, err
+	var storeInstance store.Store
+	if storeParams != nil {
```

Comment:
what happens if storeParams is nil? Should we return an error?

Reply:
I'll probably do that and handle in the caller.